### PR TITLE
[V2] chore: Create v2.0.0-beta.6 release

### PR DESCRIPTION
**Pull request by landreasyan**
**What type of PR is this?**
Creating a new release with scheduler extender performance optimizations. 

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: charts/index.yaml [hunk 1]
```diff
@@ -12,7 +21,7 @@ entries:
     version: v2.0.0-beta.6
```
Comment (edreed) [inline charts/index.yaml:21]: We'll need to update the `latest-v2` chart to beta.7 as well.

**Review by sunpa93 — APPROVED** [review]
We need to update the `latest_v2` charts as well but let's keep it for a separate PR given that we are short in time and a new commit will require new test runs which will take longer to complete?

EDIT:
If helm's going to be confused of `appVersions` with `same version`, let's make a new commit.

File: charts/index.yaml (hunk 1 above)
Comment (landreasyan) [reply charts/index.yaml:21]: Oh, that's right!

**Review by sunpa93 — CHANGES_REQUESTED** [review]
`latest-v2/azuredisk-csi-driver-v2.0.0-beta.6.tgz` needs to be removed.

File: charts/index.yaml [hunk 2]
```diff
@@ -3,7 +3,25 @@ entries:
...
+    appVersion: v2.0.0-beta.6
+    created: "2022-08-11T14:09:09.585067103-07:00"
+    description: Azure disk Container Storage Interface (CSI) Storage Plugin
+    digest: 0e7280d2bcd752668c4439078ffdfc4567ac97af97d8baa07c322b99c34fb741
+    name: azuredisk-csi-driver
+    urls:
+    - https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/main_v2/charts/v2.0.0-beta.6/azuredisk-csi-driver-v2.0.0-beta.6.tgz
+    version: v2.0.0-beta.6
+  - apiVersion: v1
+    appVersion: latest-v2
```
Comment (sunpa93) [inline charts/index.yaml:23]: Probably will need to run `helm repo index` once more?

Comment (edreed) [inline charts/index.yaml:23]: There's a duplicate entry for `latest-v2`. Re-index now that the beta.6 tgz has been deleted?

Comment (landreasyan) [reply charts/index.yaml:23]: yep 🤦‍♀️

**Pull request merged**